octobot_trading/personal_data/orders/order.py

```diff
@@ -866,6 +869,18 @@ def _update_type_from_raw(self, raw_order):
             self.exchange_order_type = enums.TradeOrderType.UNKNOWN
         self.side, self.order_type = parse_order_type(raw_order)
 
+
+    def _should_instant_fill(self):
+        if self.order_type in (enums.TraderOrderType.SELL_MARKET, enums.TraderOrderType.BUY_MARKET):
+            return True
```

Comment:
Why does this code is not in market_order class?

Reply:
right, i'll change it